concepts.tex

```diff
@@ -326,7 +326,7 @@
 template <class T>
 concept bool Swappable() {
   return requires(T t, T u) {
-    swap(std::forward<T>(t), std::forward<T>(u));
+    @\newtxt{(void)}@swap(std::forward<T>(t), std::forward<T>(u));
```

Comment:
Do we have a note anywhere explaining the significance of casting to `void` wrt equality preservation?